### ISSUE TITLE
Showing PDF is password Protected

### ISSUE THREAD

**Issue by rishabhdeepsingh** [Swati4star/Images-to-PDF#444]
I try to remove pages from a PDF I created it's saying that its password protected even though I have not set any password.

How to rebuild this bug.
1. Create a new PDF. 
2. Select ```Remove Pages```
3. In remove pages select ```Select a File``` 
4. After setting a page number (e.g. ```1```  or  ```1-1```) and clicking on ```Create PDF``` I get a Toast saying PDF is password Protected even though I have not set any password.

**Comment by Swati4star** [comment]
- This issue exists when we select PDF from external storage.
- We are unable to correctly convert file path to uri.

**Comment by Cuesta46** [comment]
that could be when you created a pdf and shared you have to posible license read and write or only read

**Comment by rishabhdeepsingh** [comment]
Its an issue with Internet storage as well. I don't have any External storge.

**Comment by Swati4star** [comment]
Yes! The issue is, we are getting uri from the intent, and we are incorrectly parsing it to wrong storage path.

The pdf reader is unable to read any PDF from that path, and is throwing `password protected PDF` error

**Comment by sunnyjain** [comment]
the issue still persists  when selecting the PDF that was created earlier, in 'compress pdf' section

**Comment by Swati4star** [comment]
Is it happening if the select the PDF from the bottomsheet or using the blue button? @sunnyjain

**Comment by sunnyjain** [comment]
the blue button!

**Comment by Swati4star** [comment]
We have this issue with a lot of PDFs (if they are selected using the blue button), we are unable to correctly convert their path to uri.

**Comment by Swati4star** [comment]
We can open up a different issue for the above. 👍
